homeassistant/components/husqvarna_automower/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure:
+    status: todo
+    comment: waiting for PR to get merged
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup:
+    status: done
+    comment: We check if we can get an access token
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: no subscription required, coordinator handles callbacks
```

Comment:
I'm not sure about this.

Reply:
That's fine, there is some better wording for this out there though